src/sonic-frr/patch/series

```diff
@@ -24,6 +24,7 @@
 0041-bgpd-backpressure-Fix-to-avoid-CPU-hog.patch
 0042-zebra-Use-built-in-data-structure-counter.patch
 0043-zebra-Use-the-ctx-queue-counters.patch
+0044-zebra-Prevent-starvation-in-dplane_thread_loop.patch
```

Comment:
not too sure if this id getting duplicated will cause any potential issue, but since it is validated, I will still getting it merged.

@vivekrnv as FYI, just in case.